### PR TITLE
Add some of the missing setting to 'page_rule' api

### PR DESCRIPTION
**Pull request by grogancolin**
This adds in (I think all) of the missing Page Rule settings.

I got the list of settings from the Create Page UI in my Cloudflare dashboard, so it may differ if some are hidden from there.

### CODE REVIEW

File: page_rules.go [hunk 1]
```diff
@@ -52,26 +65,40 @@ type PageRuleAction struct {
...
+	"opportunistic_encryption":    "Opportunistic Encryption",    // Value of type string
+	"origin_error_page_pass_thru": "Origin Error Page Pass-thru", // Value of type string
+	"polish":                      "Polish",                      // Value of type string
+	"resolve_override":            "Resolve Override",            // Value of type string
+	"respect_strong_etag":         "Respect Strong ETags",        // Value of type string
+	"response_buffering":          "Response Buffering",          // Value of type string
+	"rocket_loader":               "Rocker Loader",               // Value of type string
+	"security_level":              "Security Level",              // Value of type string
+	"server_side_exclude":         "Server Side Excludes",        // Value of type string
+	"smart_errors":                "Smart Errors",                // Value of type string
```
Comment (dknecht) [inline page_rules.go]: This doesn't exist anymore.

**Comment by grogancolin** [comment]
All of them? They're available in my UI (or rather, they were when I checked last week).
Is there anywhere with a complete list of available page rule options and I get can working on correcting that!

Thanks!

File: page_rules.go (hunk 1 above)
Comment (grogancolin) [reply page_rules.go]: All of them? They're available in my UI (or rather, they were when I checked last week).
Is there anywhere with a complete list of available page rule options and I get can working on correcting that!

Thanks!

Comment (rita3ko) [reply page_rules.go]: Just Smart Errors.

**Comment by patryk** [comment]
Thanks, Colin!

**Pull request merged**